### DISCUSSION TITLE
0% volume is still audible.

### DISCUSSION THREAD

**Discussion by mzayats** [MonitorControl/MonitorControl#1563] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When setting the volume to 0% the volume gets very low, but not fully muted.

### Steps to reproduce

Use the app controls or keyboard "volume down" or keyboard "mute" button.

### Expected behavior

Volume should be fully muted

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.0.1
- Mac model: MacBook Pro (14-inch, 2021)
- MonitorControl version: 4.2.0 Build 7048
- Monitor(s): Samsung C34J79x
- Apple Silicon/M1 (yes or no): yes
```

**Comment by LiWinDom** [comment]
[This fix](https://github.com/MonitorControl/MonitorControl/pull/1465) works for me somehow

**Comment by LiWinDom** [comment]
I was able to compile this, hope it will work
[MonitorControl.zip](https://github.com/MonitorControl/MonitorControl/files/15336177/MonitorControl.zip)

**Comment by waydabber** [comment]
Thanks @LiWinDom . :)

Also, there are various other mute/unmute options in BetterDisplay, see this: https://github.com/waydabber/BetterDisplay/issues/1248
